### PR TITLE
Fix bulk place info remote mixer data merge issue

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: internal/server/handler_v1.go [hunk 1]
```diff
@@ -130,19 +130,43 @@ func (s *Server) BulkPlaceInfo(
...
-			"/v1/bulk/info/place",
-			in,
-			remoteResp); err != nil {
-			return nil, err
+	keyedInfo := map[string]*pbv1.PlaceInfoResponse{}
+	for _, item := range localResp.Data {
+		keyedInfo[item.GetNode()] = item
+	}
+	if s.metadata.RemoteMixerDomain != "" {
+		in.Nodes = []string{}
```
Comment (pradh) [inline internal/server/handler_v1.go:138]: Add some comment?

**Review by pradh — APPROVED** [review]
Thanks!

File: internal/server/handler_v1.go (hunk 1 above)
Comment (shifucun) [reply internal/server/handler_v1.go:141]: Done

**Comment by shifucun** [comment]
thanks!

**Pull request merged**